app/formats/document_types.yml

```diff
@@ -129,6 +129,13 @@
     hostname: whitehall-admin
     path: /government/admin/news/new
 
+- id: government_response
+  label: Government response
+  description: Government statements in response to media coverage, such as rebuttals and ‘myth busters’
```

Comment:
Where does this come from? It's not in @benhazell's [document types document](https://docs.google.com/document/d/1ivZ34_SBeIRllwi31MVewyxudE7F0QnW3U3p5QoHZ6E/edit).

Reply:
This comes directly from Whitehall - if you create a new 'News' document, this is one of the options and therefore one of the kinds of document we're importing under 'NewsArticle'.

I can split this into a separate PR if you want - just trying to bung all the fixes in one.

Reply:
Ben's doc is the authoritative design of these pages, so if something is not on there you can assume it's for good reason. I wouldn't add this, but you could talk to Ben if you this there's been a mistake.

Reply:
We're not supporting the creation of government response document type. 
I think @huminmiah wanted it cut based on usage data and @paola-roccuzzo  research.

Reply:
No probs - cut it.